websites/M/Musixmatch/dist/metadata.json

```diff
@@ -21,6 +21,7 @@
 		"curators.musixmatch.com"
 	],
 	"version": "2.0.1",
+>>>>>>> edd2595e328a8c1f8c69a4b31f15b48ec10772c7
```

Comment:
You should undo the changes to this file and `yarn.lock` and re-push. The PR should only contain changes to your 4 files in `websites/W/watchmovieshd`